Vienna/Sources/Parsing/XMLFeed.m

```diff
@@ -0,0 +1,125 @@
+    @"EEE, dd MMM yy HH:mm:ss XXXX",  @"EEE, dd MMM yyyy HH:mm:ss XXXX",
+    @"EEE, dd MMM yy HH:mm:ss zzz",   @"EEE, dd MMM yyyy HH:mm:ss zzz",
+    // 2010-09-28T17:31:25+0200
+    @"yy-MM-dd'T'HH:mm:ssXXXX",       @"yyyy-MM-dd'T'HH:mm:ssXXXX",
+    // 2010-09-28T15:31:25.815+02:00
+    @"yy-MM-dd'T'HH:mm:ss.SSSXXXX",   @"yyyy-MM-dd'T'HH:mm:ss.SSSXXXX",
+    @"yy-MM-dd'T'HH:mm:ss.SSSXXXXX",  @"yyyy-MM-dd'T'HH:mm:ss.SSSXXXXX",
+    // Fri, 12 Dec 2008 18:45:15 -08:00
+    @"EEE, dd MMM yy HH:mm:ss XXXXX", @"EEE, dd MMM yyyy HH:mm:ss XXXXX",
+    @"EEE, dd MMM yy HH:mm:ss",       @"EEE, dd MMM yyyy HH:mm:ss",
+    // Other exotic and non standard date formats
+    @"yy-MM-dd HH:mm:ss XXXX",        @"yyyy-MM-dd HH:mm:ss XXXX",
+    @"yy-MM-dd HH:mm:ss XXXXX",       @"yyyy-MM-dd HH:mm:ss XXXXX",
+    @"yy-MM-dd HH:mm:ss zzz",         @"yyyy-MM-dd HH:mm:ss zzz",
+    @"EEE dd MMM yy HH:mm:ss zzz",    @"EEE dd MMM yyyy HH:mm:ss zzz",
+    @"EEE dd MMM yy HH:mm:ss XXXX",   @"EEE dd MMM yyyy HH:mm:ss XXXX",
+    @"EEE dd MMM yy HH:mm:ss XXXXX",  @"EEE dd MMM yyyy HH:mm:ss XXXXX",
+    @"EEE dd MMM yy HH:mm:ss",        @"EEE dd MMM yyyy HH:mm:ss",
+    @"EEEE dd MMMM yy",               @"EEEE dd MMMM yyyy",
+    @"dd MMM yy HH:mm:ss zzz",        @"dd MMM yyyy HH:mm:ss zzz",
+    @"dd MMM yy HH:mm:ss XXXX",       @"dd MMM yyyy HH:mm:ss XXXX",
+    @"dd MMM yy HH:mm:ss XXXXX",      @"dd MMM yyyy HH:mm:ss XXXXX",
+};
+
+static size_t const kNumberOfDateFormatters = sizeof(kDateFormats) / sizeof(kDateFormats[0]);
+
+/// C array of NSDateFormatters: creating a NSDateFormatter is very expensive,
+/// so we create those we need early in the program launch and keep them in
+/// memory.
+static NSDateFormatter *dateFormatterArray[kNumberOfDateFormatters];
+
+@implementation VNAXMLFeed
+
+// MARK: Initialization
+
++ (void)initialize
+{
+    if (self != [VNAXMLFeed class]) {
+        return;
+    }
+
+    // Initializes the date formatters
+    NSLocale *locale = [NSLocale localeWithLocaleIdentifier:@"en_US_POSIX"];
+    NSTimeZone *timeZone = [NSTimeZone timeZoneForSecondsFromGMT:0];
+
+    for (NSInteger i = 0; i < kNumberOfDateFormatters; i++) {
+        dateFormatterArray[i] = [[NSDateFormatter alloc] init];
+        dateFormatterArray[i].locale = locale;
+        dateFormatterArray[i].timeZone = timeZone;
+        dateFormatterArray[i].dateFormat = kDateFormats[i];
+    }
+}
+
+// MARK: Public methods
+
+- (void)identifyNamespacesPrefixes:(NSXMLElement *)element
+{
+    self.rdfPrefix = [element resolvePrefixForNamespaceURI:@"http://www.w3.org/1999/02/22-rdf-syntax-ns#"];
+    if (!self.rdfPrefix) {
+        self.rdfPrefix = @"rdf";
+    }
+
+    self.mediaPrefix = [element resolvePrefixForNamespaceURI:@"http://search.yahoo.com/mrss/"];
+    if (!self.mediaPrefix) {
+        self.mediaPrefix = @"media";
+    }
+
+    self.encPrefix = [element resolvePrefixForNamespaceURI:@"http://purl.oclc.org/net/rss_2.0/enc#"];
```

Comment:
this one is dead too - I'll open a new PR with fixes for this. I only mentioned it here because I noticed it.